bigtable-dataflow-parent/bigtable-beam-import/src/test/generate_test_data.txt

```diff
@@ -0,0 +1,107 @@
+create 'test', 'cf', {SPLITS => ["1", "2", "3", "4", "5", "6", "7", "8", "9"]}
```

Comment:
Please add some docs as to how someone can use this

Reply:
Adding a README.md